x-pack/plugins/synthetics/common/rules/alert_actions.test.ts

```diff
@@ -268,7 +268,7 @@ describe('Alert Actions factory', () => {
           dedupKey: expect.any(String),
           eventAction: 'resolve',
           summary:
-            'The alert for the monitor {{context.monitorName}} checking {{{context.monitorUrl}}} from {{context.locationName}} is no longer active: {{context.recoveryReason}}.',
+            'The alert for {{context.monitorName}} ({{context.locationName}}) {{context.monitorType}} monitor is no longer active: {{context.recoveryReason}}.\nThe latest error received was: "{{{context.lastErrorMessage}}}".\n{{{context.linkMessage}}}',
```

Comment:
Should I put the monitorName in quotes @florent-leborgne

Reply:
My first suggestion would be to bold it, if not possible quotes are good as well. It could also be good to add some light formatting between the monitor name, location and type so the different key characteristics read more easily, for example:
- The alert for **Elastic.co sign up - Asia South - Single page browser monitor, is no longer active:
- The alert for **Elastic.co sign up, Asia South - Single page browser monitor is no longer active:

Reply:
I went with `The alert for "Elastic.co sign up", Asia South - Single page browser monitor, is no longer active`